nbodykit/base/particles.py

```diff
@@ -119,9 +148,8 @@ def attrs(self):
         except AttributeError:
             self._attrs = {}
             return self._attrs
-
-    @staticmethod
-    def compute(*args, **kwargs):
+            
+    def compute(self, *args, **kwargs):
         """
         Our version of :func:`dask.compute` that computes
         multiple delayed dask collections at once
```

Comment:
perhaps. or maybe just a warning? It is sometimes nice when the data is small and you have a single dask array just to call .compute(). Also, shall we change the behavior so compute() does not return tuples of length one (this seems pretty annoying)?